src/Microsoft.Health.Fhir.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -68,11 +69,11 @@ public ActionResult CurrentVersion()
         [HttpGet]
         [AllowAnonymous]
         [Route(KnownRoutes.Script, Name = RouteNames.Script)]
-        public ActionResult SqlScript(int id)
+        public FileContentResult SqlScript(int id)
         {
             _logger.LogInformation($"Attempting to get script for schema version: {id}");
-
-            throw new NotImplementedException(Resources.ScriptNotImplemented);
+            string fileName = $"{id}.sql";
+            return File(Encoding.ASCII.GetBytes(ScriptProvider.GetMigrationScript(id)), "application/json", fileName);
```

Comment:
We might want to look into having a version of `GetMigrationScript` that returns something that doesn't require the conversion to text and then to bytes.

Reply:
Addressed it.